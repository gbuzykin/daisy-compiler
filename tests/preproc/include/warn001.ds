
   #   include  "pass001.dsh" 1