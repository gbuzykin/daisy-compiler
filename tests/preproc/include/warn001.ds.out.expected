./preproc/include/warn001.ds:2:31: warning: extra tokens at end of preprocessing directive
 2 |    #   include  "pass001.dsh" 1
   |                               ^
In file included from ./preproc/include/warn001.ds:2
./preproc/include/pass001.dsh:1:1: debug: token
 1 | const b = 2;
   | ^~~~~
In file included from ./preproc/include/warn001.ds:2
./preproc/include/pass001.dsh:1:7: debug: id: b
 1 | const b = 2;
   |       ^
In file included from ./preproc/include/warn001.ds:2
./preproc/include/pass001.dsh:1:9: debug: token
 1 | const b = 2;
   |         ^
In file included from ./preproc/include/warn001.ds:2
./preproc/include/pass001.dsh:1:11: debug: integer number: 2
 1 | const b = 2;
   |           ^
./preproc/include/pass001.dsh:1:7: debug: defining constant `b`
 1 | const b = 2;
   |       ^
In file included from ./preproc/include/warn001.ds:2
./preproc/include/pass001.dsh:1:12: debug: token
 1 | const b = 2;
   |            ^
In file included from ./preproc/include/warn001.ds:2
In file included from ./preproc/include/pass001.dsh:5
./preproc/include/pass001-1.dsh:2:4: debug: token
 2 |    const c = 3 ;
   |    ^~~~~
In file included from ./preproc/include/warn001.ds:2
In file included from ./preproc/include/pass001.dsh:5
./preproc/include/pass001-1.dsh:2:10: debug: id: c
 2 |    const c = 3 ;
   |          ^
In file included from ./preproc/include/warn001.ds:2
In file included from ./preproc/include/pass001.dsh:5
./preproc/include/pass001-1.dsh:2:12: debug: token
 2 |    const c = 3 ;
   |            ^
In file included from ./preproc/include/warn001.ds:2
In file included from ./preproc/include/pass001.dsh:5
./preproc/include/pass001-1.dsh:2:14: debug: integer number: 3
 2 |    const c = 3 ;
   |              ^
./preproc/include/pass001-1.dsh:2:10: debug: defining constant `c`
 2 |    const c = 3 ;
   |          ^
In file included from ./preproc/include/warn001.ds:2
In file included from ./preproc/include/pass001.dsh:5
./preproc/include/pass001-1.dsh:2:16: debug: token
 2 |    const c = 3 ;
   |                ^
./preproc/include/warn001.ds: info: warnings 1, errors 0
